src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterArrayDeclarations.java

```diff
@@ -43,4 +43,10 @@ int get1(int k, int c, int b) [] { //Incorrect
     private long getLongMultArray(int someParam, String value) [][][] { //Incorrect
         return null;
     }
+    int aa = new int[]{1}[0];//Correct
+    int bb = new int[]{1}[ 0];//Incorrect
+    int aaa = new int[][]{{1},{2}}[0][0];//Correct
+    int bbb = new int [][]{{1},{2}}[0][0];//Incorrect
+    int ccc = new int[] []{{1},{2}}[0][0];//Incorrect
+    int ddd = new int[][]{{1},{2}}[0] [0];//Incorrect
```

Comment:
To be added:
`new int[][]{{1},{2}} [0][0]`
`new int[][]{{1},{2}}[ 0][0]`
`new int[][]{{1},{2}}[0 ][0]`
`new int[][]{{1},{2}}[0][ 0]`
`new int[][]{{1},{2}}[0][0 ]`

Reply:
I added next tests:
`int in1 = new int[][]{{1},{2}}[ 0][0];//Correct`
`int in2 = new int[][]{{1},{2}}[0 ][0];//Correct`
`int in3 = new int[][]{{1},{2}}[0][ 0];//Correct`
`int in4 = new int[][]{{1},{2}}[0][0 ];//Correct`